examples/organization-viewer-cli/src/index.ts

```diff
@@ -134,7 +134,7 @@ async function inspectVotingHighLevel(appAddress: string): Promise<void> {
   console.log(voting.toString())
 
   console.log('\nVotes:')
-  const votes = await voting.votes()
+  const votes = await voting.votes(2, 1)
```

Comment:
Hmm, for some reason I naturally expect the `first` and `skip` params to be parts of an `options` object, e.g: `await voting.votes({ 2, 1})`. Not uncomfortable with the current implementation, but just a thought!